src/parser/formatting-evaluator-module.ts

```diff
@@ -40,6 +41,11 @@ export class FormattingEvaluatorModule extends BaseModule {
 
   constructor(context: ContextPlugin) {
     super(context);
+    this._readabilityConfig = this._configuration?.readabilityScoring ?? {
+      enabled: true,
+      weight: 0.3,
+      idealScore: 60,
```

Comment:
Isn't 60 kind of low? How did you come to this number @shiv810

Reply:
"60.0–50.0	10th to 12th grade	Fairly difficult to read." In the context of the other descriptions I think this is the best choice, yes.